spec/features/saml/loa1_sso_spec.rb

```diff
@@ -41,17 +41,17 @@
       expect(page).to_not have_css('.accordion-header')
     end
 
-    it 'allows user to view recovery code via profile', :js do
+    it 'user cannot view recovery code via profile', :js do
```

Comment:
you are right! I think the description can use some improvement. What about

`user cannot view profile during sign up process` ?


I think that since the logic changed, I can add two specs out of this one: 

1. spec for confirming recovery code during sign up (with JS turned on, since we didnt have a test for that before I added this one in a recent PR)
1. spec for confirming that the profile and sign out link do not show during sign up process

sound good?